index.js

```diff
@@ -99,14 +90,14 @@ SmartBanner.prototype = {
 		sb.className = 'smartbanner smartbanner-' + this.type;
 
 		sb.innerHTML = '<div class="smartbanner-container">' +
-							'<a href="javascript:void(0);" class="smartbanner-close">&times;</a>' +
+							'<a href="javascript:void(0);" class="smartbanner-close" data-atlas="smartbanner close">&times;</a>' +
```

Comment:
@jbbuckner can we look at setting these some other way? maybe listen for a click on these buttons in our Outlook code and then calling the same function that's called on anything that has `data-atlas` on it?